src/proxy-ui-api/frontend/vue.config.js

```diff
@@ -3,7 +3,7 @@ const path = require('path');
 
 module.exports = {
   devServer: {
-    proxy: process.env.PROXY_ADDRESS || 'https://localhost:4100',
+    proxy: process.env.PROXY_ADDRESS || 'https://localhost:5005',
```

Comment:
Is this change intentional?

Reply:
No.. Needs to be fixed. Leaked due to local port mapping setup.

Reply:
fixed in : 1159ab7